CHANGELOG.md

```diff
@@ -110,4 +110,24 @@ All notable changes to this project will be documented in this file. The format
 ### Changed
   - AKO library has been upgraded. User have to update CRD definitions and AKO to 1.11.1 before upgrading AMKO to 1.11.1.
 
+## AMKO-1.12.1
+
+### Added:
+  - AMKO now claims support for Kubernetes 1.28, 1.29 and OCP 4.14
+
+### Changed:
+  - PKI Profile used in GSLB Site Persistence can be added via `GlobalDeploymentPolicy` and `GSLBHostRule`
+
+## AMKO-1.13.1
+
+### Added:
+  - AMKO now claims support for Kubernetes 1.31 and OCP 4.17
+  - AMKO now support multitenancy at namespace level. This feature allows AMKO to map each namespace in Kubernetes/OpenShift cluster uniquely to a tenant in Avi.
+  - AMKO now supports OpenShift Routes with `spec.subdomain` field specified instead of `spec.host` field, using the `defaultDomain` field in GDP object. The default domain specified is appended to `spec.subdomain` to form the FQDN for the GS.
```

Comment:
Do we have doc to explain how this works? Then we can have just one liner `AMKO now supports OpenShift Routes with `spec.subdomain` field. and then point out to article with example about how it works?

Reply:
updated